Add login troubleshooting guide (#54)
conveyal/docs: docs/troubleshooting.mdx

---
title: Troubleshooting
---

import Image from '../components/image'

The following guides may help if you are having trouble using Conveyal. If you are still having problems, please email us at [support@conveyal.com](mailto:support@conveyal.com).

## Logging In

> When using Conveyal, always go to [https://analysis.conveyal.com](https://analysis.conveyal.com). 

Reviewer: ```suggestion
When using Conveyal, always navigate to [https://analysis.conveyal.com](https://analysis.conveyal.com) in your browser's address window. 
```

:::note
Bookmark [https://analysis.conveyal.com](https://analysis.conveyal.com) itself, do not bookmark the page with the login form or attempt to go to that login page directly. It may not work.
:::

If you are not logged in, the application should redirect you to a login page that looks like: 

<Image src='/img/login-page.png' alt='Log In Page' />

### Forgot password

If you've forgotten your password and would like to change it, first ensure that you have access to the email address associated with the account. Then, go to the application home page and:

1. If you are logged in, click `Log Out` at the bottom.
2. From the Log In form, click the `Don't remember your password?` link.
3. Enter the email address associated with your account. Click `Send Email`.

> 4. Follow the steps in the email, entering a new password for your account.

Reviewer: ```suggestion
4. Follow the steps in the email (check spam folders if you do not see it), entering a new password for your account.
```

5. Go to the application home page and log in with the new password.

### Other issues

If you are seeing `Unauthorized` errors or you are unable to see data for your account then you may be attempting to access a resource that you don't have access to. Another possibility is your browser may be maintaing a session in another tab that has already timed out. 

Before contacting support, we recommend trying to log into Conveyal using private browsing or incognito mode. To enter a private browser session in your browser, follow the key combinations found [here](https://en.wikipedia.org/wiki/Private_browsing#Support_in_popular_browsers). If you are still having trouble logging in while using a private browsing mode, please contact [support](mailto:support@conveyal.com).

## Server status

> If you are having unexplained problems using our application, please check our [Status Page](https://analysis.conveyal.com/status) to verify that the issue is not stemming from connectivity or log in issues. The page should show two green `OK`s and there should be no error status bars at the top. Like this:

Reviewer: ```suggestion
If you are having unexplained problems using our application, please check our [Status Page](https://analysis.conveyal.com/status) to verify that the issue is not stemming from connectivity or log in issues. The page should show two green `OK`s, without any error status bar:
```
